Dockerfile

```diff
@@ -32,3 +32,5 @@ FROM scratch
 COPY --from=stage1 /target /
 RUN xbps-reconfigure -a && \
   rm -r /var/cache/xbps
+
+CMD ["/bin/dash"]
```

Comment:
This should be /bin/sh, if you make that change I'll merge.

Reply:
Thoughts: when we use Void Linux on bare metal or ssh to VM, the default shell (in stock setting) is `/bin/dash`, and user can chose to switch to `sh`. Should we be consistent with that experience?

Reply:
You should be aware that /bin/sh is a symlink to /bin/dash.

Reply:
Ops, sorry I didn't realized that /bin/sh is a symlink.